include/sycl_ext_complex.hpp

```diff
@@ -1232,6 +1240,436 @@ _SYCL_EXT_CPLX_INLINE_VISIBILITY complex<_Tp> tan(const complex<_Tp> &__x) {
 
 _SYCL_EXT_CPLX_END_NAMESPACE_STD
 
+////////////////////////////////////////////////////////////////////////////////
+// MARRAY IMPLEMENTATION
+////////////////////////////////////////////////////////////////////////////////
+
+_SYCL_BEGIN_NAMESPACE
+
+// marray of complex class specialisation
+template <typename T, std::size_t NumElements>
+class marray<sycl::ext::cplx::complex<T>, NumElements> {
+private:
+  using DataT = sycl::ext::cplx::complex<T>;
```

Comment:
Can we rename DataT do `ComplexDataT` or `ComplexT` just for sake of clarity?
I Was confused at the beginning between T and dataT

Reply:
Updated the type name